project_stock/tests/common.py

```diff
@@ -22,26 +22,15 @@ def setUpClass(cls):
         cls.location = cls.picking_type.default_location_src_id
         cls.location_dest = cls.picking_type.default_location_dest_id
         cls.analytic_account = cls.env["account.analytic.account"].create(
-            {"name": "Test account"}
-        )
-        cls.analytic_tag_1 = cls.env["account.analytic.tag"].create(
             {
-                "name": "Test tag 1",
-                "active_analytic_distribution": True,
-                "analytic_distribution_ids": [
-                    (0, 0, {"account_id": cls.analytic_account.id, "percentage": 100}),
-                ],
-            }
+                "name": "Test account",
+                "plan_id": cls.env.ref("analytic.analytic_plan_projects").id,
```

Comment:
I believe that this plan should not be used, the new defined plans should be created with the corresponding distribution so that the value of the tests is the same.

Reply:
Plan is now a required field, that's why I used it https://github.com/odoo/odoo/blob/6b101d5ff0ad78b9ae2ad312f362c2224e7af457/addons/analytic/models/analytic_account.py#L39

Reply:
Ok sorry for the misunderstanding, I meant to create a new plan (`account.analytic.plan`) in test and not re-use an existing one.